recipes/gbmunge/makefile.patch

```diff
@@ -0,0 +1,30 @@
+--- Makefile.old	2019-04-24 17:17:58.618580369 -0500
++++ Makefile	2019-04-24 17:23:40.137613637 -0500
+@@ -1,5 +1,13 @@
+-all:
++PREFIX :=
++EXECUTABLE := gbmunge 
++
++all: $(EXECUTABLE)
++
++$(EXECUTABLE):
+ 	$(MAKE) -C src
+ 
+ clean:
+ 	$(MAKE) -C src clean
++
++install:
++	cp src/$(EXECUTABLE) $(PREFIX)/bin/
```

Comment:
Can't you just use `cp src/gbmunge $PREFIX/bin` instead of `make install` in the `build.sh`?